core/examples/luigi-example-js/webpack.config.js

```diff
@@ -1,24 +1,43 @@
 const CopyPlugin = require('copy-webpack-plugin');
+const TerserPlugin = require('terser-webpack-plugin');
+
+const path = require('path');
 
 module.exports = {
+  output: {
+    path: path.resolve(__dirname, 'public'),
```

Comment:
```suggestion
    path: path.resolve(__dirname, 'public'),
    filename: 'luigi-config.js'
```
Filename is required otherwise the config will go to `main.js`

Reply:
Thanks for noting, included.